torchbenchmark/models/llama/model.py

```diff
@@ -22,6 +22,8 @@ class ModelArgs:
     max_batch_size: int = 32
     max_seq_len: int = 1024
 
+    device: str = "cuda"
```

Comment:
Let's try avoid specifying the default device to cuda here. Can we set it to `None` and type to `Optional[str]`?

Reply:
Done